NDTensors/test/NDTensorsTestUtils/device_list.jl

```diff
@@ -1,11 +1,14 @@
 using NDTensors: NDTensors
+using Pkg: Pkg
 if "cuda" in ARGS || "all" in ARGS
   using CUDA
```

Comment:
I'm confused by something. I see `CUDA` wasn't in the `[extras]` section of `NDTensors/test/Project.toml`, how as it being installed?

Reply:
So CUDA is in the `[deps]` section but perhaps its a better idea in general to use `Pkg.add("CUDA")` here, that way we don't have to always compile `CUDA.jl` when we run `Pkg.test("NDTensors")`.

Reply:
Sure, let's try that.

Reply:
Okay I updated the code, thanks!